frontend/src/components/collab/chat/ChatSpace.tsx

```diff
@@ -92,7 +88,11 @@ const ChatSpace = ({ onClose }: IChatSpaceProps) => {
         {
           <ul className="space-y-3 px-4">
             {messages.map((item) => (
-              <ChatBubble message={item} isSelf={item.senderId === user.id!} />
+              <ChatBubble
+                key={crypto.randomUUID()}
```

Comment:
oh u got it to work

Reply:
I think the issue is that when we provide elements inside a `ul` tag, we need to explicitly provide a key.